modules/Template/Data.yaml

```diff
@@ -178,6 +178,10 @@ classes:
         assay:
         - RNA-seq
         - lncRNA-seq
+        - genePerturbed
```

Comment:
Hey Aditya, this is the correct template, but in our walkthrough, a similar update was made by adding attributes to under `slots:`. Remember that this assay annotation is meant for mapping, and not actually used.

Reply:
Ahh got it thank you!